test/examples_test.go

```diff
@@ -132,14 +132,6 @@ func TestAttestFnCall(t *testing.T) {
 		Run(filepath.Join(scriptDir, projectName+".txtar"))
 }
 
-func TestOnChain(t *testing.T) {
```

Comment:
I'm removing this test since it relies on `inputs/out.json` being up-to-date. The next test generates it on the fly and uses the `TA_FILE` variable to pass it around.